config/initializers/route_validators.rb

```diff
@@ -23,10 +23,14 @@ def initialize(request)
     end
 
     def match?
-      return false if locked_version_lower_than_current_version?
+      return false if locked_version_lower_than_current_version? && (HostingEnvironment.production? || HostingEnvironment.sandbox_mode?)
```

Comment:
adding a check for the environment here is stopping us blocking higher versions i.e. if I access version 1.3 I don't get an error

Reply:
Addressed.